brainstormforce/ultimate-addons-for-gutenberg
# Fix: Advanced Column - Reverse column and stack issue on tablet/mobile.

### Description

Trello - https://trello.com/c/6nLyuUal/205-fix-reverse-column-issue-on-tablet-mobile

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?

- Drag and drop the Advanced column and choose the column width 50/50.
- Place the some text on the left column and right column.
- Reverse column on a tablet - it should reverse on a tablet, not on mobile devices.
- Reverse column on mobile - it should reverse on mobile, not on tablet devices.
- Same for the stack option. It should be stack on respective devices.

### Checklist:
